src/interrogate/config.py

```diff
@@ -1,12 +1,17 @@
-# Copyright 2020 Lynn Root
+# Copyright 2020-2024 Lynn Root
 """
 Configuration-related helpers.
 """
 # Adapted from Black https://github.com/psf/black/blob/master/black.py.
 
+from __future__ import annotations
+
 import configparser
 import os
 import pathlib
+import re
+
+from typing import Any, Sequence
```

Comment:
`typing.Sequence` and the like are deprecated as part of [PEP 585](https://peps.python.org/pep-0585/), so it'd be better to use `collections.abc.Sequence` here